src/resmom_win/start_exec.c

```diff
@@ -1997,8 +1998,11 @@ finish_exec(job *pjob)
 			/* Need to launch a shell*/
 			launch_shell = 1;
 		}
+		/* If we fail to get cmd shell(unlikely), use "cmd.exe" as shell */
+		if (0 != get_cmd_shell(cmd_shell, sizeof(cmd_shell)))
```

Comment:
Thanks. It's a bargain between a good programming practice vs uniformity. On second thoughts, I am inclined to promote this practice in order to avoid hard to find bugs caused by accidental assignments. I will keep it as it is.